src/Maestro/Maestro.Data/Models/ReleasePipeline.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Maestro.Data.Models
+{
+    public class ReleasePipeline
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }
+
+        public int PipelineIdentifier { get; set; }
```

Comment:
How does this identify the release pipeline? Is it just the ID of the release pipeline within the AzDO instance?
If that's the case, then we should introduce a new table for identifying the release pipelines more thoroughly, including their account name, project name (if necessary).

Reply:
Yes, to identify+trigger the pipeline I only need the ID of the pipeline. I assumed that all pipelines would be from the same account/project that's why I only added the ID. 

I'll add the new fields and also make the relation many-to-many.